app/src/main/java/com/example/marvelapp/api/service/CharacterDetailService.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.marvelapp.api.service
+
+import com.example.marvelapp.api.util.Result
+import com.example.marvelapp.entity.Character
+
+interface CharacterDetailService {
+    fun getCharacterById(characterId: String): Result<Character>
```

Comment:
this should be in the same MarvelService that you had created

Reply:
done